app/rdf/parse-search-results.ts

```diff
@@ -5,6 +5,12 @@ import { SearchCube } from "@/domain/data";
 import { DataCubePublicationStatus } from "@/graphql/resolver-types";
 import * as ns from "@/rdf/namespace";
 
+const visualizePredicates = {
+  hasDimension: "https://visualize.admin.ch/hasDimension",
+  hasTermset: "https://visualize.admin.ch/hasTermset",
+  hasTimeUnit: "https://visualize.admin.ch/hasTimeUnit",
+};
```

Comment:
💯 Still think it could be useful to extract visualize namespace, I think there were other places there it was used 👀

Reply:
We actually already had it yes in rdf/namespace. I reused it now.